css/form.css

```
@@ -1500,6 +1523,16 @@ form hr {
   .form-group.w-1_2 .fl-time-range .fl-time-picker {
     width: 100%;
   }
+
+  .form-group.w-1_4 .fl-time-picker > .form-control,
+  .form-group.w-1_4 .fl-date-picker > .form-control {
+    padding-left: 40px !important;
+  }
+
+  [data-widget-package="com.fliplet.form-builder"] .form-group.w-1_4 .fl-time-picker > .fa,
```

Is `[data-widget-package="com.fliplet.form-builder"] ` needed if this file is already loaded after the files from `fliplet-api`?

```suggestion
  .form-group.w-1_4 .fl-time-picker > .fa,
```

---

Yeah, I removed them